requirements/edx/base.txt

```diff
@@ -35,6 +35,7 @@ django-threaded-multihost==1.4-1
 django-method-override==0.1.0
 djangorestframework==2.3.14
 django==1.4.18
+elasticsearch<1.0.0
```

Comment:
Please pin this to a specific version, as per the other libraries. Because this requirements file is used to install to production, we want exact versions to ensure consistency between prod and staging and any local development environments.